account/manager_test.go

```diff
@@ -1430,3 +1430,28 @@ func TestMakeTxnLabel(t *testing.T) {
 		require.Equal(t, genLabel, testCase.label)
 	}
 }
+
+// TestParseTxLabel tests whether an example account transaction label can be
+// parsed correctly.
+func TestParseTxLabel(t *testing.T) {
+	t.Parallel()
+
+	label := `poold -- {"account":{"action":"withdraw","key":` +
```

Comment:
nit: i think we use double quotes `"` with `+`

Reply:
I think that would mean adding `\"`, `\}`, and `\{` throughout. Which would be less readable. Should I still go with `"` for consistency?